### DISCUSSION TITLE
Dependency deduplication across multiple instances

### DISCUSSION THREAD

**Discussion by SevenOutman** [codesandbox/sandpack#260] [Q&A] [answered]
Hi, I wonder how should I prevent the same packages (e.g. react) being downloaded multiple times if I'm creating multiple `<Sandpack>` instances on a single page?

**Comment by DeMoorJasper** [comment, marked as answer]
The first time react gets fetched it should get cached and re-used on the other instances. We're working on improving this further though and the initial results are looking very promising

**Reply by SevenOutman** [reply]
I see. Thanks!
